.changeset/lovely-terms-drive.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': major
+---
+
+Remove MDX Fragment hack. This was used by `@astrojs/mdx` to access the `Fragment` component, but isn't require anymore since `@astrojs/mdx` v0.12.0.
```

Comment:
Nit but the fix was released in 0.12.1 — https://github.com/withastro/astro/blob/main/packages/integrations/mdx/CHANGELOG.md#0121

```suggestion
Remove MDX Fragment hack. This was used by `@astrojs/mdx` to access the `Fragment` component, but isn't require anymore since `@astrojs/mdx` v0.12.1.
```